akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSupervisionSpec.scala

```diff
@@ -387,70 +387,6 @@ class InterpreterSupervisionSpec extends AkkaSpec with GraphInterpreterSpecKit {
       lastEvents() should be(Set(OnNext(25))) // 1 + 4 + 20
     }
 
-    "restart when Conflate `seed` throws" in new OneBoundedSetup[Int](Seq(Conflate(
```

Comment:
Ok, now I remember what was the problem, I actually asked you about it once already, tried to do what you recommended and failed. 

The thing was that `withAttributes`, which I needed to set the supervisor, returns a `Graph`, so the actual `GraphStage` is wrapped in an anonymous object and because of that I couldn't use it together with the `OneBoundedSetup` in hte test.